### PR TITLE
Output logging

### LINKED ISSUE #10: Logging results to output terminal in vscode
Logging results to output terminal in vscode, rather than using the debug console or info popups.

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #10 

Changes proposed in this PR:

- Logging to output tab in the vs code extension

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -54,6 +60,8 @@ export async function activate(context: vscode.ExtensionContext) {
         const aquarius = new Aquarius(aquariusUrl)
         const asset = await aquarius.resolve(did)
 
+        outputChannel.appendLine(`Asset details: ${JSON.stringify(asset)}`)
```
Comment (mariacarmina) [inline src/extension.ts:63]: This also saves the output in the console? not only as a pop-up.

File: src/extension.ts [hunk 2]
```diff
@@ -216,6 +231,8 @@ export async function activate(context: vscode.ExtensionContext) {
         vscode.window.showInformationMessage(
           `Asset download successfully. Path: ${filePath}`
         )
+
+        outputChannel.appendLine(`Asset download successfully. Path: ${filePath}`)
       } catch (error) {
         console.error('Error details:', error)
```
Comment (mariacarmina) [inline src/extension.ts:237]: I think we should also append the errors cought within catch block.

**Review by mariacarmina — COMMENTED** [review]
some comments on my side, otherwise, looks good

File: src/extension.ts (hunk 1 above)
Comment (jamiehewitt15) [reply src/extension.ts:63]: Yeah. In the output tab, for example: 
image

File: src/extension.ts (hunk 2 above)
Comment (jamiehewitt15) [reply src/extension.ts:237]: Thanks for the feedback, I've made changes to this.

**Review by mariacarmina — APPROVED** [review]
LGTM

**Pull request merged**